build-tools/packages/build-cli/src/library/commands/base.ts

```diff
@@ -284,3 +289,18 @@ export abstract class BaseCommand<T extends typeof Command>
 		}
 	}
 }
+
+export abstract class BaseCommandWithBuildProject<
+	T extends typeof Command,
+> extends BaseCommand<T> {
+	private _buildProject: IBuildProject | undefined;
+
+	public getBuildProject(repoRoot?: string): IBuildProject {
```

Comment:
I suggest calling the parameter searchPath instead of repoRoot, since it's just passed to loadBuildProject.